src/decoder/mod.rs

```diff
@@ -31,12 +31,8 @@ mod vorbis;
 #[cfg(all(feature = "wav", not(feature = "symphonia-wav")))]
 mod wav;
 
-#[cfg(feature = "integer-decoder")]
 /// Output format of the decoders.
 pub type DecoderSample = Sample;
```

Comment:
we can inline this right? No worries I'll take care of it post merge.

Reply:
Yes, this can be inlined. I thought it might still be useful.